src/Sinch/SinchClient.cs

```diff
@@ -123,34 +137,36 @@ public class SinchClient : ISinch
             logger?.LogInformation("Initializing SinchClient...");
 
             IAuth auth =
-                new Auth.OAuth(keyId, keySecret, optionsObj.HttpClient, _loggerFactory?.Create<Auth.OAuth>());
+                new OAuth(keyId, keySecret, optionsObj.HttpClient, _loggerFactory?.Create<OAuth>());
             var httpCamelCase = new Http(auth, optionsObj.HttpClient, _loggerFactory?.Create<Http>(),
                 JsonNamingPolicy.CamelCase);
             var httpSnakeCase = new Http(auth, optionsObj.HttpClient, _loggerFactory?.Create<Http>(),
                 SnakeCaseNamingPolicy.Instance);
 
-            Numbers = new Numbers.Numbers(projectId, new Uri("https://numbers.api.sinch.com/"),
+            Numbers = new Numbers.Numbers(projectId, new Uri(NumbersApiUrl),
                 _loggerFactory, httpCamelCase);
+            Sms = new Sms(projectId, GetSmsBaseAddress(optionsObj.SmsRegion), _loggerFactory,
+                httpSnakeCase);
+            Conversation = new Conversation.Conversation(projectId,
+                new Uri(string.Format(ConversationApiUrlTemplate, optionsObj.ConversationRegion.Value)),
+                _loggerFactory, httpSnakeCase);
+
+            Auth = auth;
+
+            logger?.LogInformation("SinchClient initialized.");
+        }
 
+        private static Uri GetSmsBaseAddress(SmsRegion smsRegion)
+        {
             // only three regions are available for single-account model. it's eu and us. 
             // So, we should map other regions provided in docs to nearest server.
             // See: https://developers.sinch.com/docs/sms/api-reference/#base-url
```

Comment:
This comment is duplicated (but different) with the one describing the `GetSmsRegion` function.
May be removing this comment is fine;
Note for further discussion/ticket regarding to the `getSmsRegion` implementation: do we still want to automatically and silently move to another region ?

Reply:
I already have a task for that which I will plan to take in the next sprint

Reply:
(just reminder about duplicated but different comment regarding `GetSmSregion` call vs implementation)